addons/web/static/src/views/relational_model.js

```diff
@@ -2367,7 +2367,8 @@ export class DynamicGroupList extends DynamicList {
      * @returns {Promise<Group>}
      */
     async _createGroup(groupName, groupData = {}, isFolded = false) {
-        groupData = { ...groupData, name: groupName };
+        const fieldName = 'name' in this.activeFields ? 'name' : 'x_name';
```

Comment:
`Object.keys(this.activeFields).find(fname => ["name", "x_name"].includes(fname))`

Reply:
and a test

Reply:
We need to fallback on 'name' if `this.activeFields` doesn't contain one of 'name' or 'x_name'